jena-core/src/main/java/org/apache/jena/mem2/store/fast/FastArrayBunch.java

```diff
@@ -0,0 +1,190 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.jena.mem2.store.fast;
+
+import org.apache.jena.graph.Triple;
+import org.apache.jena.mem2.spliterator.ArraySpliterator;
+import org.apache.jena.util.iterator.ExtendedIterator;
+import org.apache.jena.util.iterator.NiceIterator;
+
+import java.util.ConcurrentModificationException;
+import java.util.NoSuchElementException;
+import java.util.Spliterator;
+import java.util.function.Consumer;
+import java.util.function.Predicate;
+
+/**
+ * An ArrayBunch implements TripleBunch with a linear search of a short-ish
+ * array of Triples. The array grows by factor 2.
+ */
+public abstract class FastArrayBunch implements FastTripleBunch {
+
+    private static final int INITIAL_SIZE = 4;
+
+    protected int size = 0;
+    protected Triple[] elements;
+
+    protected FastArrayBunch() {
+        elements = new Triple[INITIAL_SIZE];
+    }
+
+    public abstract boolean areEqual(final Triple a, final Triple b);
+
+    public boolean containsKey(Triple t) {
```

Comment:
```suggestion
    @Override
    public boolean containsKey(Triple t) {
```

Reply:
Fixed this and found other places, where I forgot @override.